spec/parts/linux/cloud-init/artifacts/cse_install_spec.sh

```diff
@@ -64,5 +64,12 @@ Describe 'cse_install.sh'
             The output line 2 should equal "Package \"containerd\" does not exist in $COMPONENTS_FILEPATH."
             The output line 3 should equal "mock installContainerdWithManifestJson calling"
         End
+        It 'should skip binary cleanup if SKIP_BINARY_CLEANUP is true'
```

Comment:
can we put this in a new `Describe` block specifically for testing the new function?